src/simulation/monsters/low/Skeleton.ts

```diff
@@ -0,0 +1,54 @@
+import LootTable from '../../../structures/LootTable';
+import SimpleMonster from '../../../structures/SimpleMonster';
+
+export const SkeletonTable = new LootTable({ limit: 128 })
+	.every('Bones')
+
+	.tertiary(5000, 'Skeleton champion scroll')
+
+	/* Herbs */
+	// TODO - herb drop table?
+	.add('Grimy guam leaf', 1, 6)
+	.add('Grimy marrentill', 1, 6)
+	.add('Grimy tarromin', 1, 6)
+	.add('Grimy harralander', 1, 6)
+	.add('Grimy ranarr weed', 1, 5)
+	.add('Grimy irit leaf', 1, 5)
+	.add('Grimy avantoe', 1, 3)
+	.add('Grimy kwuarm', 1, 3)
+	.add('Grimy cadantine', 1, 3)
+	.add('Grimy lantadyme', 1, 3)
+	.add('Grimy dwarf weed', 1, 3)
+
+	/* Runes/Ammunition */
+	.add('Bronze arrow', 2, 20)
+	.add('Bronze arrow', 5, 17)
+	.add('Earth rune', 3, 8)
+	.add('Air rune', 12, 9)
+	.add('Fire rune', 2, 4)
+	.add('Nature rune', 3, 4)
+	.add('Iron arrow', 1, 4)
+	.add('Steel arrow', 1, 3)
+
+	/* Weapons */
+	.add('Iron dagger', 1, 8)
+
+	/* Other */
+	.add('Coins', 2, 24)
```

Comment:
these coins seems really high weight compared to everything else and they blow past the set limit for the loot table